language/move-prover/interpreter/src/concrete/ty.rs

```diff
@@ -4,6 +4,7 @@
 use std::fmt;
 
 use bytecode::stackless_bytecode::Constant;
+use move_binary_format::file_format::TypeParameterIndex;
```

Comment:
Until now we have been trying to not have any references to file format outside of move-model, to keep those code bases decoupled. I think this has been violated on a few occasions already, but just want to mention this.

Reply:
I think I can redefine this and decouple `move_binary_format` from the interpreter.

Reply:
This is removed and also two references to `CodeOffset`.

However, we still need to keep `move_binary_format` for a while because of the need for
`move_binary_format::errors::{Location, PartialVMError, PartialVMResult, VMResult}`, in order for the stackless VM to produce the same results as the Move VM after execution.